PowerEditor/installer/nativeLang/english.xml

```diff
@@ -635,6 +645,14 @@ The comments are here for explanation, it's not necessary to translate them.
                     <Item id="44107" name="Switch to Folder as Workspace"/>
                     <Item id="44109" name="Switch to Document List"/>
                     <Item id="44108" name="Switch to Function List"/>
+                    <Item id="11002" name="Name A to Z"/>
+                    <Item id="11003" name="Name Z to A"/>
+                    <Item id="11004" name="Path A to Z"/>
+                    <Item id="11005" name="Path Z to A"/>
+                    <Item id="11006" name="Type A to Z"/>
+                    <Item id="11007" name="Type Z to A"/>
+                    <Item id="11008" name="Size Smaller to Larger"/>
+                    <Item id="11009" name="Size Larger to Smaller"/>
```

Comment:
Please add "Sort By" before all the entries. For example:
```xml
                    <Item id="11002" name="Sort By Name A to Z"/>
                    <Item id="11003" name="Sort By Name Z to A"/>
                    <Item id="11004" name="Sort By Path A to Z"/>
                    <Item id="11005" name="Sort By Path Z to A"/>
                    <Item id="11006" name="Sort By Type A to Z"/>
...
```
because these  entries name will appear in the shortcut mapper without "Sort By" menu. With "Sort By" in front of each command, users won't be confused.

Reply:
@donho Modifying english.xml not taking effect when notepad++ launches, it works only after changes back and forth the language in Preferences. updateCommandShortcuts() is triggering but it's not updating the strings. Is there any step that I'm missing?